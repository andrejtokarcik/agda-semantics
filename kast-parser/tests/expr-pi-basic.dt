expr = Pi x Set 2 Set 4
