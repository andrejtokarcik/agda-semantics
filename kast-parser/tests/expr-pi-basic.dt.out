<T>
    <k>
        .K
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <tenv>
        expr |-> Set 5
    </tenv>
    <data>
        .Map
    </data>
    <fun>
        expr |-> (Pi x Set 2 Set 4)
    </fun>
    <meta>
        .Map
    </meta>
</T>
